src/OpenTelemetry.Exporter.Jaeger/Implimentation/JaegerConversionExtensions.cs

```diff
@@ -0,0 +1,142 @@
+// <copyright file="JaegerConversionExtensions.cs" company="OpenTelemetry Authors">
+// Copyright 2018, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+namespace OpenTelemetry.Exporter.Jaeger.Implimentation
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Diagnostics;
+    using System.Linq;
+    using OpenTelemetry.Common;
+    using OpenTelemetry.Trace;
+    using OpenTelemetry.Trace.Export;
+
+    public static class JaegerConversionExtensions
+    {
+        private const long MillisPerSecond = 1000L;
+        private const long NanosPerMillisecond = 1000 * 1000;
+        private const long NanosPerSecond = NanosPerMillisecond * MillisPerSecond;
+
+        public static JaegerSpan ToJaegerSpan(this SpanData span)
+        {
+            var jaegerTags = new List<JaegerTag>();
+
+            if (span?.Attributes?.AttributeMap != null)
+            {
+                jaegerTags.AddRange(span.Attributes.AttributeMap.Select(a => a.ToJaegerTag()));
+            }
+
+            var jaegerLogs = new List<JaegerLog>();
```

Comment:
Would it be possible to have `JaegerLogs` be of type `IEnumerable<T>` instead of `List<T>`?
If so we could avoid allocating these lists when there are no tags nor logs to be added by adding `Enumerable.Empty<T>`.
I'm assuming it'll be often that spans won't have any logs or tags.

Reply:
@bruno-garcia Yeah, I was thinking about this when I finally carefully looked at the code while I was writing the unit tests. To explain why I was "thinking about this" now, this code is a port of some of the Jaeger .NET client code, that I modified to fit into the OpenTelemetry Exporter architecture. I wasn't paying attention to their exact data types, while I was cramming their client code into this architecture, but now that I've got some time to really reflect, I personally don't like their use direct use and exposure of List<T>.

I'm definitely planning on altering this.